maas/resource_maas_machine.go

```diff
@@ -68,6 +69,35 @@ func resourceMAASMachine() *schema.Resource {
 				Default:     "amd64/generic",
 				Description: "The architecture type of the machine. Defaults to `amd64/generic`.",
 			},
+			"block_devices": {
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: "A list of block devices attached to the machine.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id_path": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The ID path of the block device. This should be a path that is fixed and doesn't change depending on the boot order or kernel version.",
```

Comment:
The description here can confuse the user since this field is computed. "This should be a path ....". Most probably it was copied from `maas_block_device` resource.